staff/lab/anotha

```diff
@@ -0,0 +1,21 @@
+if [[ -z $1 ]]; then
+   echo "You want my advice? Don't play yourself"
+   echo ">> usage: anotha <n>"
+   exit 1
+elif [[ $1 -eq 1 ]]; then
+   REASON="anotha one"
+elif [[ $1 -gt 1 ]] && [[ $1 -le 5 ]]; then
+   REASON="bless up"
+elif [[ $1 -gt 5 ]] && [[ $1 -le 20 ]]; then
+   REASON="they will try to close the door on u,just open it"
+elif [[ $1 -gt 20 ]] && [[ $1 -le 100 ]]; then
+   REASON="the printer expect you to be greater and keep being great"
+elif [[ $1 -gt 100 ]] && [[ $1 -le 200 ]]; then
+   REASON="i changed... a lot"
+elif [[ $1 -gt 200 ]] && [[ $1 -le 1000 ]]; then
+   REASON="they don't want you to win. They don't want you to have the No. 1 print limit in the lab. They don't want you turn off economode. They don't want you buy new toner. And they don't want you to have that view."
+
+fi
+
+echo $REASON
+echo "y" | paper refund -p $1 -r "$REASON" `whoami`
```

Comment:
[Usually](http://mywiki.wooledge.org/BashFAQ/082) it's better to use the `$(...)` syntax for these, but here it doesn't really matter much